src/valkey-cli.c

```diff
@@ -75,9 +75,9 @@
 #define REDIS_CLI_KEEPALIVE_INTERVAL 15 /* seconds */
 #define REDIS_CLI_DEFAULT_PIPE_TIMEOUT 30 /* seconds */
 #define REDIS_CLI_HISTFILE_ENV "REDISCLI_HISTFILE"
-#define REDIS_CLI_HISTFILE_DEFAULT ".rediscli_history"
+#define REDIS_CLI_HISTFILE_DEFAULT ".valkeycli_history"
```

Comment:
Why not also rename the macros? We can remove the REDIS prefix on all of them i think.

```suggestion
#define CLI_HISTFILE_DEFAULT ".valkeycli_history"
```

Reply:
Thought will rename with all macro rename together and make as simple as related to only comments and logs. And now I changed macro as well and remaining macros will create another pr.